packages/nodejs/src/typeTranslation.ts

```diff
@@ -299,3 +418,188 @@ export function cryptographicParameters(
         genesisString: cp.genesisString,
     };
 }
+
+export function blockChainParameters(
+    params: v2.ChainParameters
+): v1.ChainParameters {
```

Comment:
Create a take in `chainParameters` as argument and define `const cp = chainParameters.parameters`

Reply:
Or just define `const v1 = params.parameters.v1` and `const v0 = params.parameters.v0`